src/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -1,58 +1,490 @@
 ---
-title: Block Syncer
+menuTitle: ChainSync
+title: ChainSync - synchronizing the Blockchain
 ---
-In order to ensure they are always on the correct latest state of the blockchain,
-a Filecoin node must continuously monitor and propagate blocks on the network.
 
-When a node receives blocks, it must also _validate_ them.
-Validation is split into two stages, syntactic and semantic.
-The syntactic stage may be validated without reference to additional data, the semantic stage requires access to the chain which the block extends.
+# What is blockchain synchronization?
 
-For clarity, we separate these stages into separate components:
-syntactic validation is performed by the {{<sref chainsync>}},
-and once collected and validated, the blocks are forwarded
-to the {{<sref chain_manager>}} which performs semantic validation and adds the blocks to the node's current view of the blockchain state.
+Blockchain synchronization ("sync") is a key part of a blockchain system.
+It handles retrieval and propagation of blocks and transactions (messages), and
+thus in charge of distributed state replication.
+**This process is security critical -- problems here can be catastrophic to the
+operation of a blockchain.**
 
-The block syncer interfaces and functions are included here, but we expand on important details below for clarity.
+# What is ChainSync?
 
-goFile block_syncer
+`ChainSync` is the protocol Filecoin uses to synchronize its blockchain. It is
+specific to Filecoin's choices in state representation and consensus rules,
+but is general enough that it can serve other blockchains. `ChainSync` is a
+group of smaller protocols, which handle different parts of the sync process.
 
-# Block reception and Syntactic Validation
+# Terms and Concepts
 
-- Called by: libp2p {{<sref chainsync>}}
-- Calls on: {{<sref chain_manager>}}
+- `LastCheckpoint` the last hard social-consensus oriented checkpoint that `ChainSync` is aware of.
```

Comment:
a link here to concept in consensus would be nice to disambiguate what could be meant by checkpoint

Reply:
spotted below in `block.id` file. Should link to there!